config/formats.js

```diff
@@ -1512,7 +1516,8 @@ exports.Formats = [
 	},
 	{
 		name: "[Gen 1] OU (tradeback)",
-
+      desc: [ Generation 1 OU with movepool additions from the Time Capsule.
+          "&bullet; <a href=\"http://www.smogon.com/articles/rby-tradebacks-ou/\">Information</a>"    
```

Comment:
the tabs before this line need to actually be tabs, you forgot a closing bracket as well. dont forget the blank line.